app/views/spree/admin/payment_methods/edit.html.haml

```diff
@@ -0,0 +1,18 @@
+- content_for :page_title do
+  = t(:editing_payment_method)
```

Comment:
I'd move these to be lazy lookups if it is not too much trouble.

Reply:
What are lazy hookups ?

Reply:
see here:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
Under Add translations
it's using nested structure not root level en.yml entries
basically use t(".editing_payment_method")
and move the en key under en.spree.admin.payment_method

Reply:
`ng-app` right ?